### ISSUE TITLE
msgflo-setup fails with timeout error with noflo-runtime-msgflo

### ISSUE THREAD

**Issue by bergie** [msgflo/msgflo#94]
msgflo-setup does things in this order:

* Start participants, wait for STDOUT (#92)
* Subscribe to `fbp` queue and wait for discovery messages

However, noflo-runtime-msgflo currently sends the discovery message only once, at start-up. This means msgflo-setup never receives it, as the `fbp` queue is subscribed only later.

**Comment by jonnor** [comment]
Yes, should subscribe right away, before starting the participants.
